src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DraftAdjudicationIntTest.kt

```diff
@@ -165,32 +166,33 @@ class DraftAdjudicationIntTest : IntegrationTestBase() {
   @Test
   fun `complete draft adjudication`() {
     prisonApiMockServer.stubPostAdjudication()
-    bankHolidayApiMockServer.stubGetBankHolidays()
+    val intTestData = IntTestData(webTestClient, jwtAuthHelper, bankHolidayApiMockServer, prisonApiMockServer)
 
-    val draftAdjudicationResponse = dataAPiHelpers().startNewAdjudication(dateTimeOfIncident = DATE_TIME_OF_INCIDENT)
-    dataAPiHelpers().addIncidentStatement(draftAdjudicationResponse.draftAdjudication.id, "test statement")
+    val firstDraftUserHeaders = setHeaders(username = IntTestData.DEFAULT_ADJUDICATION.createdByUserId)
```

Comment:
We ought to decide on a better way of describing the setup rather than this idiomatic use of IntTestData